packages/block-editor/src/components/block-inspector/index.js

```diff
@@ -48,7 +48,19 @@ function BlockInspectorLockedBlocks( { topLevelLockedBlock } ) {
 		},
 		[ topLevelLockedBlock ]
 	);
+
 	const blockInformation = useBlockDisplayInformation( topLevelLockedBlock );
+	const hasBlockStyles = useSelect(
```

Comment:
We can probably remove this, as it'll get in via https://github.com/WordPress/gutenberg/pull/64872.

Reply:
It's been merged. 🚀

Reply:
Rebased to remove this change 👍